src/content/reference/fleet-management/migration_to_cluster_api.md

```diff
@@ -0,0 +1,44 @@
+
+__Note:__ The `CAPI_MC_NAME` is the name of the management cluster (MC) where the Cluster API controllers are installed.
+
+## Recommendations for a smooth migration
+
+We also recommend increasing the size of your "master" node instance type to 2x or 3x its normal size for the duration of the migration. (e.g. `mx.large` to `mx.4xlarge`). This ensures the API server can effectively handle the load during the migration since there might only be one node to handle the traffic at certain points throughout the process.
+
+## The migration process
+
+The migration process consists of several steps:
+
+1. __Initialization:__ Necessary Kubernetes access credentials and AWS credentials are retrieved.  Vault client is created to interact with the Vault instance containing all security assets of the cluster.
+2. __Preparation:__ Migration of secrets to the CAPI management cluster, including CA certs, encryption provider secrets, and service account secrets. Migration scripts are created as secrets in the CAPI management cluster. Additionally, AWS credentials for the cluster are migrated by creating an `AWSClusterRoleIdentity` in the CAPI management cluster. Certain operations are performed to avoid conflicts during migration, such as disabling machine health check on the vintage cluster resources, scaling down the app operator for the migrated workload cluster, or cleaning up certain charts.
+3. __Stopping vintage cluster resource reconciliation:__ To avoid conflicts, vintage reconciliation is stopped by removing all `aws-operator` labels from the vintage cluster resource.
+4. __Cluster API cluster provisioning:__ Generation and application of CAPI cluster templates. A separate routine runs in the background to ensure the old load balancer remains active. The tool waits until at least one CAPI control-plane node joins the cluster and is in a Ready state. Various operations are performed to ensure a smooth transition, such as stopping control-plane components on the vintage cluster, cordoning all vintage control-planes, and deleting certain pods to speed up installation and updates.
+5. __Cleaning the vintage cluster:__ All vintage control-plane nodes are drained, vintage auto scaling groups are deleted, and all worker nodes for each node pool are drained and deleted.
+
+## Post-migration cleanup
+
+Our engineer will check that all resources and infrastructure are correctly migrated and that the new cluster is working as expected.
+
+By following this process, you can ensure a smooth and efficient migration of your Kubernetes clusters to the Cluster API. If you encounter any issues during the migration process, our team is ready to assist you.
```

Comment:
> By following this process, you

Sounds as if the customer runs something. Let's reword to say that it's done together (?).

Reply:
+1